src/features/code_actions.zig

```diff
@@ -509,6 +558,16 @@ fn getCaptureLoc(text: []const u8, loc: offsets.Loc) ?offsets.Loc {
     return .{ .start = start_pipe_position, .end = end_pipe_position };
 }
 
+fn prependSlice(list: *std.ArrayListUnmanaged(types.CodeAction), arena: std.mem.Allocator, values: []const types.CodeAction) !void {
+    const old_len = list.items.len;
+    try list.resize(arena, list.items.len + values.len);
+    var i = old_len + values.len - 1;
+    while (i >= values.len) : (i -= 1) {
+        list.items[i] = list.items[i - values.len];
+    }
+    std.mem.copy(types.CodeAction, list.items[0..values.len], values);
```

Comment:
The `insertSlice` function should provide what you need. I would suggest to remove the `prependSlice` function all together and just call `insertSlice` directly.
```zig
try list.insertSlice(arena, 0, values);
```